src/style/pre-match.less

```diff
@@ -408,6 +416,10 @@ a.back:hover {
 			outline: 0;
 			color: white;
 		}
+
+		&.disabled {
```

Comment:
Would be nice to have a grey disabled button graphic, but this just prevents the user from clicking "refresh" again until the current refresh has completed.